### DISCUSSION TITLE
Can't access pages using chrome browser

### DISCUSSION THREAD

**Discussion by tamirat-tade** [selenide/selenide#2248] [General]
Unable to establish websocket connection to http://localhost:53082/devtools/browser/648fcbc2-f1df-432b-a673-12d973d58785

**Comment by tamirat-tade** [comment]
anyone can help me

**Comment by asolntsev** [comment]
@tamirat-tade Isn't it the same problem as https://github.com/selenide/selenide/issues/2193 and https://github.com/selenide/selenide/issues/2192?

then you just need to upgrade to latest Selenide version.
